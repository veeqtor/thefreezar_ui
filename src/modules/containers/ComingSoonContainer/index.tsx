/** @jsx jsx */

import * as React from 'react';
import { jsx } from '@emotion/core';
import { HeroWrapper, Overlay, HeroTextWrapper } from 'modules/containers/ComingSoonContainer/index.style';
import SEO from 'modules/components/SEO';

export interface IComingSoonPageProps {
  title: string;
}

const ComingSoonContainer = ({ title }: IComingSoonPageProps): React.ReactElement => (
  <HeroWrapper>
    <SEO title={title} />
    <Overlay>
      <HeroTextWrapper>
        <p>Coming soon</p>
        <h1>The Freezar Nigeria</h1>
        <p>
          The freezar is a multi-media company that specializes in photography, cinematography, make-up, hair and
          content creation.
        </p>
        <span></span>
      </HeroTextWrapper>
    </Overlay>
  </HeroWrapper>
);

export default ComingSoonContainer;
