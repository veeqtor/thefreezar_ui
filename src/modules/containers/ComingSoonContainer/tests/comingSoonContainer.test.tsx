/* eslint-disable @typescript-eslint/explicit-function-return-type */
/* eslint-disable @typescript-eslint/no-explicit-any */

import * as React from 'react';
import { render, cleanup } from '@testing-library/react';
import ComingSoonContainer from 'modules/containers/ComingSoonContainer';

const renderComingSoonPage = (props: any) => {
  return {
    ...render(<ComingSoonContainer {...props} />),
  };
};

describe('Coming soon page Test', () => {
  afterEach(cleanup);

  it('Should render', () => {
    const {
      getByText,
      container: { firstChild },
    } = renderComingSoonPage({ title: 'Coming Soon' });
    expect(getByText('The Freezar Nigeria')).toBeInTheDocument();
    expect(getByText('Coming soon')).toBeInTheDocument();
    expect(firstChild).toMatchSnapshot();
  });
});
