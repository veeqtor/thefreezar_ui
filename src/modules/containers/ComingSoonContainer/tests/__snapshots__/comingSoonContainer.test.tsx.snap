// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Coming soon page Test Should render 1`] = `
<section
  class="css-4vu9f7"
>
  <div
    class="css-yqg6lc"
  >
    <div
      class="css-7w8oqj"
    >
      <p>
        Coming soon
      </p>
      <h1>
        The Freezar Nigeria
      </h1>
      <p>
        The freezar is a multi-media company that specializes in photography, cinematography, make-up, hair and content creation.
      </p>
      <span />
    </div>
  </div>
</section>
`;
